bm_scrapy/utils.py

```diff
@@ -6,3 +6,9 @@ def parse_time(date=None):
         date = datetime.now()
     parsed_time = date.strftime("%d/%m/%Y %H:%M:%S.%f")[:-3]
     return parsed_time
+
+
+def datetime_to_json(o):
+    if isinstance(o, datetime):
+        return o.__str__()
+    raise TypeError("Type %s not serializable" % type(obj))
```

Comment:
Shouldn't this be `type(o)`?

Reply:
It was to see if you were attentive
![](https://github.githubassets.com/images/icons/emoji/unicode/1f605.png)
, just kidding. Fixed.